source/deployment/bulk-loading-data-format.rst

```diff
@@ -538,6 +538,32 @@ For clarity, the object is shown using regular JSON formatting, but in the data
       "teams": [
         {
           "name": "team-name",
+          "theme": [
```

Comment:
Are the ``image`` and ``type`` fields used to show the image in Account Settings > Display > Theme?